_topic_maps/_topic_map.yml

```diff
@@ -993,6 +993,17 @@ Topics:
   Distros: openshift-enterprise,openshift-origin
 - Name: Using PTP hardware
   File: using-ptp
+- Name: AWS Load Balancer Operator
+  Dir: aws_load_balancer_operator
+  Topics:
+  - Name: Understanding AWS load balancers
```

Comment:
I'm not sure we should be documenting AWS features in OpenShift docs. The sections in the enhancement was to help the reviewers of the enhancement. We should not have it in the official docs.

Reply:
Addressed